integration/release/release.go

```diff
@@ -0,0 +1,61 @@
+// +build k8srequired
+
+package release
+
+import (
+	"log"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+)
+
+func WaitForReleaseStatus(gsHelmClient *helmclient.Client, release string, status string) error {
```

Comment:
These functions were moved to reuse them in a new test.

Reply:
Nice move to make these functions reusable for future tests :+1: 
About the naming: Since we have it in package `release`, For me would be fine to call it `release.WaitForStatus()` to not have the double release. But no strong opinions here.

Reply:
+1 I have a strong opinion on this.

Reply:
And done.